docs/datasource_api_userguide.md

```diff
@@ -242,73 +260,82 @@ CREATE TABLE tpch_test.TARGET_TABLE_CUSTOMER (
-| spark.tispark.pd.addresses  | -             | true     | PD Cluster Addresses, split by comma                                                                        | -       |
-| spark.tispark.tidb.addr     | tidb.addr     | true     | TiDB Address, currently only support one instance                                                           | -       |
+| spark.tispark.pd.addresses  | -             | true     | The addresses of PD clusters, split by comma                                                                        | -       |
+| spark.tispark.tidb.addr     | tidb.addr     | true     | TiDB address, which currently only supports one instance                                                           | -       |
 | spark.tispark.tidb.port     | tidb.port     | true     | TiDB Port                                                                                                   | -       |
 | spark.tispark.tidb.user     | tidb.user     | true     | TiDB User                                                                                                   | -       |
 | spark.tispark.tidb.password | tidb.password | true     | TiDB Password                                                                                               | -       |
 | database                    | -             | true     | TiDB Database                                                                                               | -       |
 | table                       | -             | true     | TiDB Table                                                                                                  | -       |
-| skipCommitSecondaryKey      | -             | false    | skip commit secondary key                                                                                   | false   |
-| enableRegionSplit           | -             | false    | do region split to avoid hot region during insertion                                                        | true    |
-| regionSplitNum              | -             | false    | user defined region split number during insertion                                                           | 0       |
-| replace                     | -             | false    | define the behavior of append.                                                                              | false   |
-| lockTTLSeconds              | -             | false    | tikv lock ttl, write duration must < `lockTTLSeconds`, otherwise write may fail because of gc               | 3600    |
-| writeConcurrency            | -             | false    | maximum number of threads writing data to tikv, suggest `writeConcurrency` <= 8 * `number of TiKV instance` | 0       |
+| skipCommitSecondaryKey      | -             | false    | To skip commit secondary key                                                                                   | false   |
```

Comment:
What does it mean by "skip commit secondary key"? looks like it makes no sense grammatically.

Reply:
whether to skip the commit phase of secondary keys.